### ISSUE TITLE
Support Scala Native for Scala 3

### ISSUE THREAD

**Issue by lbialy** [VirtusLab/scala-yaml#324]

**Comment by WojciechMazur** [comment]
> I think this is limited to changing sbt cross-compilation configuration, isn't it @WojciechMazur?

Yes, probably. I don't see any blockers in dependencies. If there are some lacks in Java Standard Library coverage then (anyone who's going to add SN support) let me know
